### PR TITLE
Add new authors: Alex and Ivan

### PR DESCRIPTION
**Pull request by Totktonada**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -1 +1,3 @@
 Alexander Turenko <alexander.turenko@tarantool.org, totktonada.ru@gmail.com>
+Alex Khatskevich <avkhatskevich@tarantool.org>
```
Comment (Khatskevich) [inline AUTHORS]: Please, add avkhatskevich@gmail.com :)

Comment (Totktonada) [reply AUTHORS]: Done.

**Review by SudoBobo — APPROVED** [review]
It is all fine

File: AUTHORS (hunk 1 above)
Comment (Khatskevich) [reply AUTHORS]: Ack

**Review by Khatskevich — APPROVED** [review]

**Pull request merged**